packages/core/src/odata/common/filter/filter-lambda-expression.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { EntityBase } from '../entity';
+import { FieldType } from '../selectable';
+import { Filter } from './filter';
+import { Filterable } from './filterable';
+
+/**
+ * @experimental This is experimental and is subject to change. Use with caution.
+ */
+export type FilterLambdaOperator = 'any' | 'all';
+
+/**
+ * @experimental This is experimental and is subject to change. Use with caution.
+ */
+export class FilterLambdaExpression<
+  FieldT extends FieldType,
+  LinkedEntityT extends EntityBase = any
+> {
+  constructor(
+    public rootFiledName: string,
+    public innerFilter: Filter<LinkedEntityT, FieldT>,
+    public lambdaOperator: FilterLambdaOperator
+  ) {}
+}
+
+// eslint-disable-next-line valid-jsdoc
```

Comment:
is this needed? For the js docs above it is not there.

Reply:
I guess it only complains if there are function arguments etc missing? SO then it is fine.

Reply:
Without this, there will be this lint error:
```
  26:1  error  Missing JSDoc @returns for function       valid-jsdoc
  26:1  error  Missing JSDoc for parameter 'filterable'  valid-jsdoc

```